### ISSUE TITLE
Confusing notifications when ejecting a USB drive that is busy

### ISSUE THREAD

**Issue by BillDietrich** [linuxmint/nemo#2172]
```
nemo 4.0.6 on Mint 19.1 Cinnamon
 ```

**Issue**

When you copy a lot of files to a USB drive, then immediately try to eject the drive in Nemo, first you get a "wait, don't unplug the drive yet, still writing" notification, then an "okay, now it's safe to unplug the drive" notification.  The issue is that the two notifications look identical; they're both yellow "caution" notifications.  It's very easy to glance at the first "wait" notification and assume it is the second "okay" notification, and unplug the drive, leading to data loss.

Please change the notifications to be clearer.  Maybe the first should have a red "stop" icon, and the second should have a green "okay" icon ?  I'm not sure what choices are available in the UI.  Thanks.

**Comment by nmjbhoffmann** [comment]
Agreed, I made this mistake a couple times when I first stopped using it. These days I run the command "sync", then when it stops running I know the files actually copied, write caching on memory sticks are a real pain.
But yeah, having something distinct would be nice. An animation on the drive icon showing it's still loading like a spinning wheel when ejecting or something is one idea.

**Comment by BillDietrich** [comment]
Why is this not higher priority and fixed ?  It should be a simple fix, just change colors in existing notifications.  This problem causes a risk of data loss.

**Comment by icarter09** [comment]
@BillDietrich can you post a screen capture of the two notifications? Thanks.

**Comment by BillDietrich** [comment]
Sorry, I stopped using Mint a while ago.

Should be easy enough to replicate.

**Comment by Jeremy7701** [comment]
Is this an issue in the current version of nemo [4.6.5] ?

**Comment by Bugfollower** [comment]
Identical point of view here: it is very dangerous, happened also to me, and red and green colours are the best idea.
Happy to see that anyone on earth has the same idea.
Few people reported this here, but it happened to a lot of others.
Always think of the poor old grandmother discovering the linux world, and avoid her ending in tears.

While copying a big file to an external disk, I right clicked on the corresponding disk partition shown on the left panel of nemo and clicked again to ask for ejecting the disk while the copy was not yet finished.
Then I made 2 screenshots where the two different message panels appeared, with white characters on a black background.
Then with GIMP I changed the colours of the panels, the first in red, the second in green, to show you the wished suggestion.
Here are the 4 pictures, the first two ones are the real screenshots, and the last two ones are not real, but the modified pictures with GIMP, as suggestions.
The first two ones are confusing, because sometimes the user does not read the text in the panel, wrongly thinking everything is already OK, because in most cases only the panel "OK" appears.
In the last two ones the colours prevent from any confusion.
nemo 4.8.4

![2021 01 21 22 26 20 Périphérique avant](https://user-images.githubusercontent.com/75504873/105427204-eb4e9e00-5c4c-11eb-9c30-897a5727008a.png)
![2021 01 21 22 41 07 Périphérique après](https://user-images.githubusercontent.com/75504873/105427210-ec7fcb00-5c4c-11eb-8d5c-b6e34a22d3be.png)
![2021 01 21 23 26 20 Périphérique rouge](https://user-images.githubusercontent.com/75504873/105427211-edb0f800-5c4c-11eb-93fa-a014889096b2.png)
![2021 01 21 23 41 07 Périphérique vert](https://user-images.githubusercontent.com/75504873/105427218-eee22500-5c4c-11eb-9571-63ca7491a588.png)

**Comment by icarter09** [comment]
@Bugfollower thank you for the explanation and screen shots! Much appreciated and exactly what I was looking for.

**Comment by Jeremy7701** [comment]
Read the screen before removal - not after removal.

**Comment by LinuxOnTheDesktop** [comment]
> Read the screen before removal - not after removal.

That misses the point: Mint aims to make Linux a less effortful thing. Further: there is no advantage to leaving the messages as they stand. Further yet: even the following small change would help: one capitalises a relevant word ('do NOT remove one's device') within one of the messages.

**Comment by AttilaLL** [comment]
and it takes too long between them, if the second even appears - copying 16 gb of files to USB stick is excercise in frustration, Windows does not have problem and shows real file copy progress on USB and then you can unplug it in seconds so you know how much you have to wait.. I am just too tired for this ####... choosing between Microslop spyware and nerdOS that is outside of reality .. is just too much. The absolute state of modern software...
